src/frontend/scss/objects/_form.scss

```diff
@@ -83,23 +83,13 @@
     position: relative;
 
     &:hover {
```

Comment:
I check the hover effect on storybook. Indeed, for text inputs and select, just fade the border color is enough.

But about radio and checkbox inputs, on hover, I'll also fade the label color (but not the message color).